ldappool/__init__.py

```
@@ -0,0 +1,450 @@
+            else uri
+        )
+        self.uri = uri if isinstance(uri, LDAPUrl) else LDAPUrl("ldap:///")
+        self.binddn = binddn
+        self.bindpw = bindpw
+        self.params = params if params is not None else {}
+        self.max = int(max)
+        self._lock = threading.Lock()
+        self._pool = []
+        logging.debug("ConnectionPool %s starting with %s connections", self, self.max)
+        if self.params.get("prewarm", False):
+            self.scale
```

Ruff (0.14.8)

Found useless expression. Either assign it to a variable or remove it.

(B018)